rmgpy/pdep/network.py

```diff
@@ -327,7 +327,7 @@ def set_conditions(self, T, P, ymB=None):
 
         n_isom = self.n_isom
         n_reac = self.n_reac
-        n_proc = self.n_prod
+        n_prod = self.n_prod
```

Comment:
This is indeed a typo, but `n_prod` (or `n_proc`...) isn't used in this function. Should we remove this line instead?

Reply:
We could remove it in this PR, though I do use it in #1810. If it is removed in this PR, I'll just add it back in the other one.